### ISSUE TITLE
Fail to push work from my editors terminal

### ISSUE THREAD

**Issue by Gideon-Daniels** [microsoft/vscode#137223]
Issue Type: <b>Bug</b>

remote: No anonymous write access.
fatal: Authentication failed for 'https://github.com/Gideon-Daniels/Vue-Muso-Ninja.git/'

VS Code version: Code 1.62.2 (3a6960b964327f0e3882ce18fcebd07ed191b316, 2021-11-11T20:51:30.084Z)
OS version: Linux x64 5.11.0-40-generic snap
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Celeron(R) N4020 CPU @ 1.10GHz (2 x 2700)|
|GPU Status|2d_canvas: unavailable_software<br>gpu_compositing: disabled_software<br>multiple_raster_threads: disabled_off<br>oop_rasterization: disabled_off<br>opengl: disabled_off<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: unavailable_software<br>webgl2: unavailable_software|
|Load (avg)|1, 1, 1|
|Memory (System)|3.64GB (0.12GB free)|
|Process Argv|--no-sandbox --force-user-env --unity-launch --crash-reporter-id f1bd9400-a49a-40c3-b26f-6489e93b26a4|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|wayland|
</details><details><summary>Extensions (6)</summary>

Extension|Author (truncated)|Version
---|---|---
prettier-vscode|esb|9.0.0
auto-save|mcr|1.0.3
vetur|oct|0.35.0
material-icon-theme|PKi|4.10.0
LiveServer|rit|5.6.1
vsfire|tob|1.4.1

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30384386
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
vsccppwtct:30382698
bridge0708:30335490
dockerwalkthru:30377721
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsqsis200:30386379
vsaa593cf:30376535
vssld246cf:30386378

```

</details>

**Comment by faresdjefaflia** [comment]
**You will find the solution to the problem here**
https://github.com/microsoft/vscode/issues/137161#issuecomment-968833263

**Comment by meganrogge** [comment]
/duplicate of #137183 which is scheduled to be fixed in upcoming recovery release (1.62.3)
